liferay-workspace/apps/service-builder/basic/basic-web/src/main/resources/META-INF/resources/edit_foo.jsp

```diff
@@ -40,9 +40,9 @@ if (fooId > 0) {
 		title='<%= (foo != null) ? foo.getField1() : "new-foo" %>'
 	/>
 
-	<liferay-ui:asset-categories-error />
+	<liferay-asset:asset-categories-error />
```

Comment:
Is this something new in 7.2?

Reply:
We should likely have some rule in upgrade tool for it if so.

Reply:
I did a search in liferay-portal for `asset-categories-error` and found that this is how they changed it. But there are other things that do not render in this jsp.